frontend/src/metabase/dashboard/components/RefreshWidget/RefreshWidget.module.css

```diff
@@ -0,0 +1,47 @@
+.RefreshWidgetPopover {
+  padding: 1rem;
+  min-width: 12.5rem;
+}
+
+.RefreshWidgetTitle {
+  color: var(--color-text-medium);
+  font-weight: bold;
+  font-size: 0.75em;
+  text-transform: uppercase;
+  margin-bottom: 1em;
+  margin-left: 0.5em;
+}
```

Comment:
I'm curious on when do we decide to write our own styles in CSS Modules, as opposed to using `metabase/ui` Mantine components? (Mantine components can specify most of these props)

Reply:
I think the main blocker is nested selectors, as you see later in the file. I've moved the independent classes to the mantine props as you suggested, but some of them will have to stay for now

Reply:
Got it! I think it makes complete sense to go with CSS modules for nested selectors, as there is no other way to handle it 😄 

Mantine's style props is much more limited than, say, what tailwind and friends can do unfortunately